H5P.BranchedVideo/branched-video.js

```diff
@@ -779,6 +781,76 @@ H5P.BranchedVideo = (function ($) {
       settingsDiv.style.display = 'none';
       rightControls.appendChild(settingsDiv);
 
+      // playback speed
+      var speedButton = document.createElement('button');
+      speedButton.type = 'button';
+      speedButton.style.top = '-15px';
+      speedButton.id = 'tapestry-speed-button';
+      var speedButtonText = document.createTextNode('Speed');
+      speedButton.appendChild(speedButtonText);
+      settingsDiv.appendChild(speedButton);
+
+      var speedDiv = document.createElement('div');
+      speedDiv.className = 'tapestry-settings-dropdown-content';
+      speedDiv.style.display = 'none';
+      rightControls.appendChild(speedDiv);
+
+      var speed1 = document.createElement('button');
+      speed1.style.top = '-30px';
+      var speed1Text = document.createTextNode('0.5');
+      speed1.appendChild(speed1Text);
+      speedDiv.appendChild(speed1);
+
+      var speed2 = document.createElement('button');
+      speed2.style.top = '-15px';
+      var speed2Text = document.createTextNode('Normal');
+      speed2.appendChild(speed2Text);
+      speedDiv.appendChild(speed2);
+
+      var speed3 = document.createElement('button');
+      speed3.style.top = '0px';
+      var speed3Text = document.createTextNode('1.5');
+      speed3.appendChild(speed3Text);
+      speedDiv.appendChild(speed3);
+
+      var speed4 = document.createElement('button');
+      speed4.style.top = '15px';
```

Comment:
Could this be moved into the CSS by any chance?

Reply:
I could move these to CSS but I would have a separate class for each speed button, would that be better than leaving them in here?

Reply:
Could you use `:nth-child()` pseudo-class?

Reply:
I will try it out thanks!